### PR TITLE
Epic 2/story current building as start 6

### PR DESCRIPTION
**Pull request by david0455**
- Determine if user is in a campus building

image

Hardcoded coordinates to Hall Building to show that the feature works. (since we cannot go out):
image

### CODE REVIEW

File: SD-K9/src/app/components/outdoor-map/outdoor-map.component.ts [hunk 1]
```diff
@@ -288,4 +300,29 @@ export class OutdoorMapComponent implements OnInit, OnDestroy, Map {
...
+      isInside = google.maps.geometry.poly.containsLocation(coordinates, polygonArr[i]);
+      if(isInside){
+        building = polygonArr[i].currentBuilding;
+        break;
+      }
+    }
+
+    if(isInside){
+      //display you are inside a campus building
+      document.getElementById('btn').innerHTML = "You are in " + building + " building";
```
Comment (a-paggabao) [inline SD-K9/src/app/components/outdoor-map/outdoor-map.component.ts:322]: Just a tip, you don't have to do this if you don't want.
If ever in the future you need to build a string with multiple variables u can also use this:
```suggestion
      document.getElementById('btn').innerHTML = `You are in ${building} building`;
```

Comment (paulMoose) [inline SD-K9/src/app/components/outdoor-map/outdoor-map.component.ts:322]: The angular way of doing this is using [Template syntax](https://angular.io/guide/template-syntax#interpolation-and-template-expressions). You can have a variable message = 'something' and just change the message, and in the HTML file, you would have <h3 id="btn">{{message}}</h3> so you don't need to deal with getting the element by id and changing the inner HTML

File: SD-K9/src/app/components/outdoor-map/outdoor-map.component.ts [hunk 2]
```diff
@@ -288,4 +302,29 @@ export class OutdoorMapComponent implements OnInit, OnDestroy, Map {
     this._unsubscribeGoogleStore();    // release listener to google-store
   }
 
+  
+
+  inCampus(coordinates){ 
+    //coordinates = new google.maps.LatLng(45.497307, -73.578971);   //to test Hall Building coordinates
```
Comment (ThierryRomain) [inline SD-K9/src/app/components/outdoor-map/outdoor-map.component.ts:308]: please remove if not useful

File: SD-K9/src/app/pages/location-search/location-search.page.spec.ts [hunk 3]
```diff
@@ -63,7 +63,13 @@ describe('LocationSearchPage', () => {
     expect(nav.navigateBack).toHaveBeenCalledWith("home");
   });
 
+<<<<<<< HEAD
+  it('#changeQuery should filter elements in `itemList`', async() => {
+    await delay(20000);
+
+=======
   it('#changeQuery should filter elements in `itemList`', () => {
+>>>>>>> master
```
Comment (ThierryRomain) [inline SD-K9/src/app/pages/location-search/location-search.page.spec.ts:72]: What is this? please resolve conflict before merging

**Pull request merged**